### ISSUE TITLE
RandomBrightness giving black/white images

### ISSUE THREAD

**Issue by SachinKarmani** [albumentations-team/albumentations#1156]
## 🐛 Bug

Albumentations version ( 1.1.0)
Python version (3.8.8)
OS ( Windows)
Tensorflow (2.5.0)

I was using albumentation for Keras ImageDataGenerator and my accuracy was not crossing 25%. Upon further investigation, I saw that the augmentation pipeline was giving black and white images,even though I have used a very low value of brightness.

    def augmentor_dummy(image, img_size):
        transformations = A.Compose([  A.RandomBrightness(limit=0.001) ])    
        data = {"image":image}
        aug_data = transformations(**data)
        aug_img = aug_data["image"]
        aug_img = tf.cast(aug_img/255.0, tf.float32)
        aug_img = tf.image.resize(aug_img, size=[img_size, img_size])
        return aug_img

    train_datagen = ImageDataGenerator(preprocessing_function = augmentor_dummy)

    imageGen = train_datagen.flow_from_directory(r'C:\Users\sachin.kirmani\Downloads\stuff\TopTal CV\temples-train-hard\\',
                                             batch_size=1, 
                                             save_to_dir=r'C:\Users\sachin.kirmani\Downloads\stuff\TopTal CV\augment_output\\',
                                             save_prefix="image",
                                             target_size=(256,256) )

    total = 0
    for image in imageGen:
        total += 1
        if total == 100:
            break

![Capture](https://user-images.githubusercontent.com/52692163/163733702-961ab309-baa2-4d17-9b85-ba5e94fb0320.PNG)

**Comment by HAL41** [comment]
I believe RandomBrightness and RandomBrightnessContrast work only with float32 and not uint8. Therefore add `A.ToFloat(max_value=255)` as the first step in A.Compose and now you can skip line 5` aug_img = tf.cast(aug_img/255.0, tf.float32)`

```
def augmentor_dummy(image, img_size):
    transformations = A.Compose([
        A.ToFloat(max_value=255),
        A.RandomBrightness(limit=0.001)
    ])    
    aug_img = transformations(image=image)["image"]
    aug_img = tf.image.resize(aug_img, size=[img_size, img_size])
    return aug_img
```

**Comment by SachinKarmani** [comment]
Thanks, will try this. I did not think of this as I did not see this ToFloat function be used before RandomBrightness in any documentation/example.

**Comment by ternaus** [comment]
Everything works now: https://explore.albumentations.ai/transform/RandomBrightnessContrast
